### PR TITLE
January 2020 slides

### PR DESCRIPTION
**Pull request by WyriHaximus**
- [ ] Add next hoster information
- [ ] Remove PHP benelux
- [ ] Add next speaker info

### CODE REVIEW

File: 2020-January.html [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+                    "hoster-small": {
+                        "src": "img/hoster-logos/castor.svg",
+                        "alt": "Hosted by Castor"
+                    },
+                    "hoster-large": {
+                        "src": "img/hoster-logos/castor.svg",
+                        "alt": "Hosted by Castor"
+                    },
+                    "hoster-next": {
+                        "src": "img/hoster-logos/castor.svg",
```
Comment (lcobucci) [inline 2020-January.html]: Next hoster isn't Castor, right?

File: 2020-January.html [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+                        "alt": "Castor"
+                    },
+                    "speaker-face": {
+                        "src": "img/speakers/thijs_de_maa.jpeg",
+                        "alt": "Speaker: Thijs de Maa"
```
Comment (lcobucci) [inline 2020-January.html]: Need to update the next speaker too

File: 2020-January.html [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+                        </div>
+                    </div>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Conferences and Events</h1>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">PHP Benelux</h1>
```
Comment (lcobucci) [inline 2020-January.html]: Wasn't this conf sold out? If so, please remove it.

File: 2020-January.html (hunk 1 above)
Comment (WyriHaximus) [reply 2020-January.html]: Couldn't find the next one on the sheet. But didn't have time yet to ping slack :P

File: 2020-January.html (hunk 2 above)
Comment (WyriHaximus) [reply 2020-January.html]: This is this months speaker AFAIK

File: 2020-January.html (hunk 3 above)
Comment (WyriHaximus) [reply 2020-January.html]: On my todo list to check

File: 2020-January.html (hunk 2 above)
Comment (lcobucci) [reply 2020-January.html]: You're right, sorry

File: 2020-January.html (hunk 3 above)
Comment (WyriHaximus) [reply 2020-January.html]: They stopped selling tickets yesterday actually, so I've removed it

File: 2020-January.html (hunk 1 above)
Comment (WyriHaximus) [reply 2020-January.html]: Dropped the next hoster for now

**Pull request merged**